### ISSUE TITLE
CPU use only efficiency core 

### ISSUE THREAD

**Issue by VASAPOL** [iperov/DeepFaceLab#5450]
Hello,
I recently upgrade my computer from i5 9400f to i9 12900k
before I upgrade(i5 9400f)
deepfacelab using my cpu around 100% and after I upgrade to i9 deep face use efficiency core and not use performance core.
![Screen Shot 2564-12-28 at 21 18 23](https://user-images.githubusercontent.com/43716227/147590995-67cab221-3d37-496c-87ac-20afcff226f1.png)

I tried to update the version of deep face and issue found again.

Window 10 Pro

**Comment by dexter-pl** [comment]
I don't think it's a problem with DeepFaceLabs but with Windows or CPU. I use a lot of python processes and also have E-cores on full and P-cores almost nothing.
Have you figured it out?

**Comment by ghost** [comment]
Did you ever find the answer? If so, would you mind sharing it and closing this issue?

**Comment by dexter-pl** [comment]
No, I haven't. I haven't used my multi-process app after that, so I don't know whether it's fixed. I guess the e-cores were relatively new back then so maybe drivers weren't quite ready.
